src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Meta, StoryObj } from '@storybook/react'
+
+import { Checkbox } from '.'
+import { defaults } from './Checkbox'
+
+// NOTE: we are not using the `actions` addon here to simulate the onClick and onChange props
+// because it is causing glitches in the switch animation.
```

Comment:
this was occurring in the switch, as stated in the copy-pasted (😛) comment, does this issue occur also for the checkbox? if so please update the comment according to the proper component

Reply:
right... done.